### DISCUSSION TITLE
Question: do we need a privacy policy or kind of a similar statement?

### DISCUSSION THREAD

**Discussion by Oliver-Loeffler** [gluonhq/scenebuilder#615] [Q&A]
I was looking into various ways to distribute Scene Builder from this repository. Interesting places are e.g. the Microsoft Store and the Apple store but there are plenty others such as Chocolatey or FlatPak etc.

Some of those distribution channels require a link to a privacy policy.
As far as I know, Scene Builder allows to register ones email address for update notifications or similar.
Besides that, its not storing any data outside the users environment.

Is there any document where one could refer to?
I personally would consider this as helpful.

What other opinions exist on this?

**Comment by AlmasB** [comment]
Yeah, would be good to clarify this in case any data is being collected.

**Comment by Oliver-Loeffler** [comment]
Scene Builder sends at least following details from the user to `http://usage.gluonhq.com/ul/log`:

1. Operating system (system properties `os.arch`, `os.name` and `os.version`), concatenated into one string,
2. Java version used (system property `java.version`),
3. The users e-mail if registered,
4. If the user subscribed or not,
5. A type (either `Tracking.SCENEBUILDER_TYPE` or `Tracking.SCENEBUILDER_USAGE_TYPE`)
6. The Scene Builder registration hash, see (`PreferencesRecordGlobal.getRegistrationHash()`)
7. The Scene Builder version.

At least the e-mail is kind of personal data. The use case is valid but it should be made transparent.
That this data is collected and sent to gluonhq.com is not stated anywhere (at least I havent found a place). 

Not sure if there is more.

**Reply by Oliver-Loeffler** [reply]
The `RegistrationWindowController` generates a unique identifier based on the systems network interface mac address. In case the id cannot be generated based on the network interface mac, a random UUID is generated. The good thing is, the mac is not directly used its always hashed.
